.github/workflows/multinode.yml

```diff
@@ -362,16 +362,28 @@ jobs:
 
       - name: Break on failure
         run: |
-          sleep ${{ env.break_duration }}m
-        if: ${{ failure() && steps.config_ach.outcome == 'success' && contains(fromJSON('["failure", "always"]'), env.break_on) }}
+          sleep ${{ inputs.break_duration }}m
+        if: ${{ failure() && steps.config_ach.outcome == 'success' && contains(fromJSON('["failure", "always"]'), inputs.break_on) }}
 
       - name: Break on success
         run: |
-          sleep ${{ env.break_duration }}m
-        if: ${{ steps.config_ach.outcome == 'success' && contains(fromJSON('["success", "always"]'), env.break_on) }}
+          sleep ${{ inputs.break_duration }}m
+        if: ${{ steps.config_ach.outcome == 'success' && contains(fromJSON('["success", "always"]'), inputs.break_on) }}
 
       - name: Destroy
-        run: terraform destroy -auto-approve -input=false
+        run: |
+          for attempt in $(seq 3); do
+              if terraform destroy -auto-approve -input=false -lock-timeout=100s; then
```

Comment:
I might be tempted to wait a little longer - there's not much to lose.

Reply:
Very true, lock timeout is now 200, sleep is 120, and now using 5 retries